newIDE/app/src/ProjectManager/index.js

```diff
@@ -738,6 +761,20 @@ export default class ProjectManager extends React.Component<Props, State> {
                 )
             }
           />
+          {!!this.state.name && (
+            <SceneEditor
```

Comment:
This is still duplicated in SceneEditor/index.js

Reply:
@4ian Changed it to use the Scene Editor directly. Don't really understand the current duplication you referring to ):. Could you a little more detail about what you meant?

Reply:
Oops sorry wrong line commented! :)

Reply:
I'm not sure actually to understand why you're putting the entire `SceneEditor` inside ProjectManager 😬
We for sure don't want to do that.

Reply:
I actually used `VariablesEditorDialog` in my first PR. But then you mentioned "duplication of code and modifying SceneEditor so that there is only one VariablesEditorDialog used for this specific case". I thought what you meant was modifying the `SceneEditor` to suit the use case for `ProjectManager` ):

Reply:
Ok I understand better :) 
So `VariablesEditorDialog` is the proper component to use. 

But we want to avoid having to have it both in ProjectManager *and* in SceneEditor if possible. While it's not a huge duplication, there is still some duplicated props like the messages to show on screen (`emptyExplanationMessage`, etc...).

Two possibilities:
* You create a SceneVariablesEditorDialog component, that pass the props that are common like the messages (`emptyExplanationMessage` and co) and just take a few less props. You then use it in ProjectManager and SceneEditor
* OR you put SceneVariablesEditorDialog in the ancestor of SceneEditor and ProjectManager which is... MainFrame. You add a a state to designate which scene (also called layout) has its variables edited. You then pass a prop to SceneEditor and ProjectManager that is a function to call to open the scene variables.

Advantage of first solution: we're not bloating MainFrame.
Advantage of second solution: we don't need a common `SceneVariablesEditorDialog`, because we only have a single place in the code where we use VariablesEditorDialog for a scene - so no duplication of `emptyExplanationMessage` and other props.

Usually I would prefer the second solution, but here because I'd like to avoid bloating too much MainFrame, maybe the first solution is better.